### DISCUSSION TITLE
SEB 3.x :- Access to webcam and audio capturing devices are blocked by Antivirus

### DISCUSSION THREAD

**Discussion by Tanmaya-Code-Library** [SafeExamBrowser/seb-win-refactoring#269] [Q&A] [answered]
I have installed SEB 3.2.0 (OS: Windows 10). The antivirus installed on my machine is blocking the access to webcam and audio capturing devices. It is not allowing CefSharp.BrowserSubprocess.exe to access camera and microphone. Please suggest how to resolve this issue.

Antivirus Name:- Kaspersky Endpoint Security for Windows 
Antivirus Version:- 11.4.0.233

Antivirus Report:-
Host Intrusion Prevention was triggered	CefSharp.BrowserSubprocess	Blocked	Security settings access	Access to webcam	High	Exactly	Access to webcam		Access to webcam	

Host Intrusion Prevention was triggered	CefSharp.BrowserSubprocess	Blocked	Security settings access	Access to audio capturing devices High	Exactly	Access to audio capturing devices Access to audio capturing devices

**Comment by dbuechel** [comment, marked as answer]
Thanks for reporting the issue. Are you using the default configuration of your anti-virus software, or have you specifically set some strict(er) settings? This is the first time we hear of this specific issue, although we unfortunately had various issues in the past where anti-virus software has interfered with SEB.

**Reply by dbuechel** [reply]
As a quick workaround, you could try to add SEB to the list of trustworthy applications in your anti-virus software or simply deactivate it temporarily while you are using SEB.

**Reply by Tanmaya-Code-Library** [reply]
I have not specially set any settings of my anti-virus software. Kindly provide a solution for this issue. So that I can run SEB 3.2 without making any changes to the anti-virus software settings.

**Reply by dbuechel** [reply]
There have been a few occasions where anti-virus software would falsely detect SEB as malicious (false positive detection). There most likely is nothing we can do about it, the fastest and most reliable way to allow the execution of SEB would be to manually add it to the list of trustworthy applications, as already explained above.
